x/posts/client/cli/cli_test.go

```diff
@@ -879,3 +879,77 @@ func (s *IntegrationTestSuite) TestCmdAnswerPoll() {
 		})
 	}
 }
+
+func (s *IntegrationTestSuite) TestCmdMovePost() {
+	val := s.network.Validators[0]
+	testCases := []struct {
+		name      string
+		args      []string
+		shouldErr bool
+		respType  proto.Message
+	}{
+		{
+			name: "invalid subspace id returns error",
+			args: []string{
+				"", "1", "2", "1",
```

Comment:
We should add the `--from` flag to all these tests in order to make sure that they don't fail due to the invalid signer address. Otherwise we will not catch errors like the one highlighted above